src/opf/acp.jl

```diff
@@ -1,293 +1,167 @@
 struct ACOPF <: AbstractFormulation end
 
 """
-    build_acopf(data, optimizer)
+    build_opf(ACOPF, data, optimizer)
 
-Build an AC-OPF model.
-
-This implementation is based on the AC-OPF formulation of Rosetta-OPF
-    https://github.com/lanl-ansi/rosetta-opf/blob/38a951326df3156d79dcdc49c8010aa29905b05d/jump.jl
+Build an ACOPF model.
 """
-function build_opf(::Type{ACOPF}, data::Dict{String,Any}, optimizer;
+function build_opf(::Type{ACOPF}, network::Dict{String,Any}, optimizer;
     T=Float64,    
 )
-    # Cleanup and pre-process data
-    PM.standardize_cost_terms!(data, order=2)
-    PM.calc_thermal_limits!(data)
-    ref = PM.build_ref(data)[:it][:pm][:nw][0]
-
-    # Check that slack bus is unique
-    length(ref[:ref_buses]) == 1 || error("Data has $(length(ref[:ref_buses])) slack buses (expected 1).")
-    i0 = first(keys(ref[:ref_buses]))
+    # TODO: remove when all formulations are done
+    data = OPFData(network)
```

Comment:
I suppose this will be changed in a later PR?

Reply:
Yes, otherwise I'd have to duplicate changes on each formulation PR so tests would still run. Each branch has this at the top and at the bottom of `build_opf` -- once all are merged, I plan to push a commit that will only store `data::OPFData` in `OPFModel`, not `network::Dict`, and the function signature for `build_opf` will change similarly